railties/lib/rails/generators/rails/benchmark/benchmark_generator.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require "rails/generators/named_base"
+
+module Rails
+  module Generators
+    class BenchmarkGenerator < NamedBase
+      argument :reports, type: :array, default: []
```

Comment:
Let's find a simple clearly named 2-value default here for the common case AB-benchmark, such that people don't have to pass those. Maybe "before", "after"?